### PR TITLE
Completed react-function-components exercise - Dulce Herrera

### PR DESCRIPTION
**Pull request by dulceherrera**
Screen Shot 2022-08-31 at 2 49 04 PM

### CODE REVIEW

File: react-function-components/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```
Comment (scbowler) [inline react-function-components/src/index.jsx]: Here you need to specify `client` for `ReactDOM`
```JAVASCRIPT
import ReactDOM from 'react-dom/client';
```

File: react-function-components/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+function CustomButton() {
+  return <button type='button'>Click Me!</button>;
+}
+
+ReactDOM.render(
+  <CustomButton/>,
+  document.querySelector('#root')
+);
```
Comment (scbowler) [inline react-function-components/src/index.jsx]: This is the old way of doing this, you need to be using the new way for React 18.
```JSX
const container = document.querySelector('#root');
const root = ReactDOM.createRoot(container);

root.render(<CustomButton />);
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but you are using an older method of rendering the element, needs to be updated to current standards. Let me know if you have any questions.

File: react-function-components/src/index.jsx (hunk 1 above)
Comment (dulceherrera) [reply react-function-components/src/index.jsx]: Screen Shot 2022-08-31 at 3 20 41 PM

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**